Task2&3/WdlV1_1Ast.py

```diff
@@ -0,0 +1,4 @@
+class TaskNode():
```

Comment:
Lets move the WDL parser/lexer into its own folder. We don't need the Java code versions, so those can be deleted